src/main/rx/srxl2.c

```diff
@@ -0,0 +1,545 @@
+#include "platform.h"
```

Comment:
This file should have the same header as https://github.com/betaflight/betaflight/pull/8606/files#diff-2da41031eb7d7f076567ebda42e5223e

Reply:
Done